### PR TITLE
Change to create 2 layers of ref portfolio

### PR DESCRIPTION
**Pull request by andrewjlp**
# Pull Request Checklist

- [x ] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

#Description of the PR

Details on Jira here: https://finbourne.atlassian.net/browse/SE-2485

Add a multi layer portfolio look through to the notebook: sample-notebooks/examples/use-cases/valuation/Look-through valuation (multi-level).ipynb

Add: GlobalEquity → GlobalEquityFinancials -> USEquityFinancials

### CODE REVIEW

File: examples/use-cases/valuation/Look-through valuation (multi-level).ipynb [hunk 1]
```diff
@@ -2,9 +2,35 @@
  "cells": [
```
Comment (cscialino) [inline examples/use-cases/valuation/Look-through valuation (multi-level).ipynb:2]: <p>Why do we multiply the units by 2 here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/529/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (cscialino) [inline examples/use-cases/valuation/Look-through valuation (multi-level).ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    transaction_data
</pre><p>Should we be showing transactions in the 3.5 Quotes section here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/529/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (andrewjlp) [reply examples/use-cases/valuation/Look-through valuation (multi-level).ipynb:2]: Not sure, that baffled me as well - it was in the original notebook.

Comment (andrewjlp) [reply examples/use-cases/valuation/Look-through valuation (multi-level).ipynb:2]: I have removed this doubling

Comment (andrewjlp) [reply examples/use-cases/valuation/Look-through valuation (multi-level).ipynb:2]: I will remove

**Review by cscialino — APPROVED** [review]

**Pull request merged**